src/components/data/FileBrowser.js

```diff
@@ -252,7 +252,7 @@ const BucketBrowserTable = ({
                           h(ClipboardButton, {
                             tooltip: 'Copy file URL to clipboard',
                             className: 'cell-hover-only',
-                            style: { marginLeft: '1ch' },
+                            style: { marginLeft: '1rem', display: 'inline-table' },
```

Comment:
Nit: suggest plain `inline` instead of `inline-table`.

```suggestion
                            style: { marginLeft: '1rem', display: 'inline' },
```

Reply:
Ah nice, that feels better. Just made that change